src/core/dbus-execute.c

```diff
@@ -1775,20 +2146,43 @@ int bus_exec_context_set_transient_property(
 
                 return 1;
 
-        } else if (streq(name, "SELinuxContext")) {
+        } else if (STR_IN_SET(name, "SELinuxContext", "AppArmorProfile", "SmackProcessLabel")) {
+                bool ignore;
                 const char *s;
 
-                r = sd_bus_message_read(message, "s", &s);
+                r = sd_bus_message_enter_container(message, 'r', "bs");
```

Comment:
I may mistakenly understand it, but `SELinuxContext` is defined as the following:
```
SD_BUS_PROPERTY("SELinuxContext", "(bs)", property_get_selinux_context, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```
I thought that the type given in `SD_BUS_PROPERTY()` and one used in `bus_exec_context_set_transient_property()` should be the same. Is it true?